backend/src/main/java/ai/verta/modeldb/entities/ProjectEntity.java

```diff
@@ -269,12 +272,20 @@ public void setAttributeMapping(List<AttributeEntity> attributeMapping) {
     this.attributeMapping.addAll(attributeMapping);
   }
 
-  public String getWorkspace() {
-    return workspace;
+  public Long getWorkspaceId() {
+    return workspaceId;
+  }
+
+  public void setWorkspaceId(Long workspaceId) {
+    this.workspaceId = workspaceId;
+  }
+
+  public String getLegacy_workspace_id() {
+    return legacy_workspace_id;
   }
 
-  public void setWorkspace(String workspace) {
-    this.workspace = workspace;
+  public void setLegacy_workspace_id(String workspace) {
+    this.legacy_workspace_id = workspace;
```

Comment:
Weird mix of camelCase and snake_case, no?

Reply:
Fixed